Source/WebCore/PAL/pal/PALSwift/UnsafeOverlays.swift

```diff
@@ -235,7 +235,8 @@ extension Curve25519.Signing.PrivateKey {
         if span.empty() {
             return try self.signature(for: Data.empty()).copyToVectorUInt8()
         }
-        return try self.signature(for: Data.temporaryDataFromSpan(spanNoCopy: span)).copyToVectorUInt8()
+        return try self.signature(for: Data.temporaryDataFromSpan(spanNoCopy: span))
+            .copyToVectorUInt8()
```

Comment:
Was this line feed intended?

Reply:
https://github.com/WebKit/WebKit/pull/27474#discussion_r1576409180